api/apitestcoreservice.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2019 IoTeX Foundation
```

Comment:
rename this file or move the definition into api/coreservice.go

Reply:
yes, move to api/coreservice.go